applejack/conf/products/xl-deploy.yml

```diff
@@ -8,20 +8,40 @@ dockerfiles:
 repositories:
   nexus: 'https://nexus.xebialabs.com/nexus/service/local/repositories/{repo}/content/com/xebialabs/deployit/xl-deploy/{version}/{product}-{version}-server-trial-edition.zip'
   dist: 'https://dist.xebialabs.com/customer/xl-deploy/product/{version}/{product}-{version}-server.zip'
+resources:
+  dirs:
+  - all
+  - common
```

Comment:
What's the difference between the `all` and the `common` directories here?

Reply:
all is for _all_, _common_ only for xld and xlr (not xlcli)